cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return self.__str__()
```

Comment:
More conventional and clear approach is to directly construct the full_name property from the relevant attributes, rather than relying on __str__()